drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/ActionWorkItemSetFieldPlugin.java

```diff
@@ -135,7 +135,7 @@ private boolean acceptParameterType(final PortableParameterDefinition ppd) {
             return false;
         }
 
-        final String fieldClassName = oracle.getFieldClassName(patternWrapper().getFactType(),
```

Comment:
@karreiro please have a look especially on this line. I had problems to mock properly in tests so I did this change, were there special reason for `patternWrapper()` instead of `editingWrapper()`?